build-android/cmake/README.md

```diff
@@ -0,0 +1,43 @@
+Build Extension Layers with Android CMake Plugin
```

Comment:
I don't recall what anything in `build-android/cmake` is for

Are you using this? Main reason is it has a lot of `validation layer` references and not sure if this is even needed

Reply:
not needed. I'm not sure why VVL has it.